build.gradle

```diff
@@ -73,7 +73,7 @@ configure(coreProjects) {
         google()
         mavenCentral()
         maven {
-            url 'http://oss.sonatype.org/content/repositories/snapshots'
+            url 'https://oss.sonatype.org/content/repositories/snapshots'
```

Comment:
Do we depend on any snapshots, or could this just be removed?

Reply:
I don't believe we do - it could be removed. It just had to be changed to support https as http is no longer supported.